CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contribution
```

Comment:
```suggestion
# Contributing to Web Machine Learning website
```

Good to make it explicit the scope here is the website to differentiate it from e.g. the WebNN API specification.

Reply:
Fixed

Reply:
Fixed.